pkgs/tools/security/clamav/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
   };
 
   # Flaky test, remove this when https://github.com/Cisco-Talos/clamav/issues/343 is fixed
```

Comment:
```suggestion
```

Reply:
Why would we remove this?

Reply:
That was in combination with the other comment.